docs/BROWSERSETUP.md

```diff
@@ -0,0 +1,39 @@
+# Browser Setup for Testing
+It is possible to activate browser specific settings before test execution.  
+E.g. when a special proxy setting is needed for Firefox on a System Under Test (SUT) environment under Linux which is different to the proxy setting under Windows, it is possible to create a JSON-File called **"browserSetup.json** in the directory **src/test/resources** of the actual test project. These specified settings will be used for test execution.
+
+## Example structure of the JSON-File   
+![BrowserSetup](/images/browserSetup.png)
+
+### Browser Setup Elements
+The **browserSetup.json** file consists of several different Browser Setup Elements marked blue in the screenshot.  
+Each Element consists of    
+* an arbitrary chosen element name (e.g. in screenshot "firefox-section-without-os") 
+* an optional Browser Setting entry for the platform (os - marked yellow in the screenshot). 
```

Comment:
this does not seem to match up with the figure, which has (from top to bottom)
* _Capabilities_ (**red**)
* _Browser Setting_ (**yellow**)
* _Browser Setup Element_ (**blue**)
* _Options_ (**green**)

The text refers to two blocks as being marked yellow, while no blue block is being mentioned. I would also suggest to list the blocks in the same order as they appear in the figure, and match up the blocks' captions verbatim, so its easier for readers to follow these cross-references.